ci/travis/ci.sh

```diff
@@ -157,7 +159,13 @@ test_python() {
       -python/ray/tests:test_stress_sharded  # timeout
       -python/ray/tests:test_webui
     )
-    bazel test -k --config=ci --test_timeout=600 --build_tests_only -- "${args[@]}";
+  fi
+  if [ 0 -lt "${#args[@]}" ]; then  # Any targets to test?
+    install_ray
+    # We need to set PYTHONPATH here to let Python find our pickle5 under pip install -e.
```

Comment:
1. change this comments with TODO
2. and specify this is unexpected and a short term hack to let test passes.
3. in long term, ray should import ray.cloudpickle -> pickle automatically.

Reply:
Done